### DISCUSSION TITLE
Anyway to write a voxelized LAS file to input into ArcGIS Pro?

### DISCUSSION THREAD

**Discussion by teide101** [r-lidar/lidR#617] [Q&A]
Hey everyone,

I am wondering if there is anyway I can write a voxelized LAS file that I can then input into ArcGIS Pro. 

For example, I know I can write a normalized LAS file using:

> writeLAS(las_normalized, './eastern_me_normalized.las')

I was hoping to do something similar, but with a LAS file I run the voxel_metrics tool on. Something like this:

> las_vox <- voxel_metrics(las_denoised, custom_metrics(Intensity), 1)

> writeLAS(las_vox, './las_vox.las')

Right now, the voxel_metrics tool creates  data.frame in R (which is great!) and I know I can run plot(las_vox) to view it, but I was hoping to actually view the voxelized LAS file in ArcGIS Pro to do some further analysis on.

**Comment by Jean-Romain** [comment]
Ask on [gis.stackexchange](https://gis.stackexchange.com) how to write a data.frame with XYZ into a format supported by your software. I can't help and you are unlikely to find somebody else than me here.
